pkg/cmd/cli/cmd/rsync.go

```diff
@@ -0,0 +1,381 @@
+package cmd
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+
+	"github.com/golang/glog"
+	"github.com/spf13/cobra"
+	"gopkg.in/pipe.v2"
```

Comment:
Would prefer not pulling in a pipe library if os.Pipe() works

On Thu, Sep 24, 2015 at 11:21 AM, Cesar Wong <notifications@github.com>
wrote:

> In pkg/cmd/cli/cmd/rsync.go
> <https://github.com/openshift/origin/pull/4787#discussion_r40331561>:
>
> > @@ -0,0 +1,381 @@
> > +package cmd
> > +
> > +import (
> > +	"bytes"
> > +	"fmt"
> > +	"io"
> > +	"os"
> > +	"os/exec"
> > +	"path/filepath"
> > +	"strings"
> > +
> > +	"github.com/golang/glog"
> > +	"github.com/spf13/cobra"
> > +	"gopkg.in/pipe.v2"
>
> The main reason was convenience. There was less setup required with this
> library. Not a big deal to go back to os.Pipe.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/4787/files#r40331561>.
>